### DISCUSSION TITLE
what is a block  not in docs

### DISCUSSION THREAD

**Discussion by gzqn** [Budibase/budibase#4399] [Help]
Apologies for dumb question but what is a block and why would I use them in preference to sections and containers?Blocks are  not covered in the docs

**Comment by mjashanks** [comment]
Blocks are common compositions of elements.

`Table Block` is a Data Provider + Table + filter
`Repeater Block` is a Data Provider + Repeater

It's a faster way to get something working in the design section.

Our intention here is that we will expand on these over time, for lots of scenarios - and _potentially_  allow "Ejection". I.e. Decompose the block into it's constituent components to allow for deeper editing.
